INSTALLING.md

```diff
@@ -90,39 +94,31 @@ cd tt-metal
 git submodule foreach 'git lfs fetch --all && git lfs pull'
 ```
 
-3. Set up the environment variables. For Grayskull, use:
-
-```sh
-export ARCH_NAME=grayskull
-export TT_METAL_HOME=$(pwd)
-export PYTHONPATH=$(pwd)
-```
-
-For Wormhole boards, use:
-
-```sh
-export ARCH_NAME=wormhole_b0
-export TT_METAL_HOME=$(pwd)
-export PYTHONPATH=$(pwd)
-```
-
-4. Install either from source, or from our release wheel.
+3. Install either from source, or from our release wheel.
 
 ### Option 1: From source
 
 We use CMake for our build flows.
 
+Set up the environment variables and invoke our build scripts. Note that for
+source builds, you must set these environment variables every time.
+
 ```sh
+export ARCH_NAME=<ARCH_NAME>
+export TT_METAL_HOME=$(pwd)
+export PYTHONPATH=$(pwd)
 ./build_metal.sh
 
 # If you would like an out-of-the-box virtual environment to use,
 ./create_venv.sh
 source python_env/bin/activate
 ```
 
-Note about Python environments: You do not have to use `create_venv.sh`. If you
-are less familiar with Python and its various environment tools, just use
-`create_venv.sh` as shown above and the pre-built environment.
+where `ARCH_NAME` is either `grayskull` or `wormhole_b0`, depending on your Tenstorrent device.
+
+> [!NOTE]
+>
+> Note about Python environments: You do not have to use `create_venv.sh`. If you are less familiar with Python and its various environment tools, just use `create_venv.sh` as shown above and the pre-built environment.
```

Comment:
I still need to reproduce the errors when building outside of the python virtual env. Will have it sometime tomorrow.